.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## What happened 🤔
+Describe the big picture of your changes here to communicate to team why we should accept this pull request.
+
+
+## Insight 👀
+Describe in details how to achieve the target feature/fixes, referenced documentation is welcome as well.
+
+
+## PoW (a.k.a Proof Of Work)💪
+Show us the proof that it worked (photo, gif...etc)
+
+<img src="URL_GOES_HERE" width=200 />
```

Comment:
Let's use the template from [here](https://github.com/nimblehq/git-template/blob/main/.github/PULL_REQUEST_TEMPLATE.md). This git template was updated.

Reply:
@lydiasama We are going to use a new template for android. I copied the template from [Trung's PR](https://github.com/nimblehq/android-templates/blob/e0307e932ba03bd85d393e8cc2c9ad70eff154cb/.githubtemplate/PULL_REQUEST_TEMPLATE.md)